Core_OverlayMods/Metadata.cs

```diff
@@ -2,7 +2,7 @@
 {
     internal static class Metadata
     {
-        public const string Version = "6.3";
+        public const string Version = "6.4";
```

Comment:
No bumping plugin version in PRs please.
```suggestion
        public const string Version = "6.3";
```